logger/logger.go

```diff
@@ -17,6 +17,15 @@
  *
  */
 
+// The logger package, when built with the structuredlogging build tag,
```

Comment:
The logger package implements opinionated logging facilities for snapd. When built ...

perhaps?